internal/provisioner/main.go

```diff
@@ -14,16 +16,13 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-package main
-
```

Comment:
Hmm. Don't we usually keep the copyright notice at the very top of the file? I'm curious if there's a specific motivation for this change.